Update Spanish translation

Vicente015 opened the pull request:
Hi, I am happy to contribute to this project again :wave: :)
I have updated the Spanish translation.

I have some words that I am not sure how to translate or what they mean so I would need some context on them. These are the following lines:

* https://github.com/Vicente015/mue/commit/11129f2d706ab61b41b40c6e930ad394b72bc3dd#diff-bedd901d303e3afce409ae5ad218fd7e2338c05c0233530846a7c4cac0728844R103
* https://github.com/Vicente015/mue/commit/11129f2d706ab61b41b40c6e930ad394b72bc3dd#diff-bedd901d303e3afce409ae5ad218fd7e2338c05c0233530846a7c4cac0728844R233

* * *

I will keep this pr as a draft until I translate the missing ones and improve some translations.

davidjcralph commented:
Hello again! Thank you for opening this pull request.

"Gaps" (couldn't think of a better name 😄) refers to the date being like "24 - 04 - 21".
"Toast duration" refers to how long the toasts (see below image) are open for.
![Example Toast](https://user-images.githubusercontent.com/14052956/115954894-5e391080-a4eb-11eb-8f8d-3fc1ae57b823.png)

Vicente015 commented:
I've added the missing translations and fixed typos.

It seems there were some problems with the analogue and language sections title was using "analog" or "language" instead of "title", I guess it wasn't changed properly with the new update, I checked and this also affected zh_CN so I fixed it.

davidjcralph commented:
Thanks!
